### PR TITLE
add alternating row colors 

### PR DESCRIPTION
**Pull request by tarkah**
## Proposed Changes
  - Row colors now alternate by default for better accessibility. This can be turned
  off in settings under the UI section's "Alternate Row Colors" checkbox.

## Checklist

- [x] Tested on Windows
- [x] Tested on MacOS
- [x] Tested on Linux
- [x] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users

### CODE REVIEW

File: src/gui/mod.rs [hunk 1]
```diff
@@ -446,6 +447,12 @@ impl Application for Ajour {
                         ExpandType::None => false,
                     };
 
+                    let is_odd = if self.config.alternating_row_colors {
+                        Some(idx % 2 != 0)
+                    } else {
+                        None
+                    };
```
Comment (casperstorm) [inline src/gui/mod.rs:454]: Clever :)

**Review by casperstorm — COMMENTED** [review]
LGTM. This I really like a lot.
I'll quickly go through all themes and adjust anything if needed.

File: src/gui/mod.rs (hunk 1 above)
Comment (tarkah) [reply src/gui/mod.rs:454]: :D

**Review by casperstorm — APPROVED** [review]

**Comment by tarkah** [comment]
> LGTM. This I really like a lot.
> I'll quickly go through all themes and adjust anything if needed.
> 
> Edit: I actually think they all look good. Not sure I want to do any updates on them. Ayu for example does blend quite a lot with bg, but still is much more readable. What do you think? I actually like it as it is.

I think it's fine. Users needing more contrast can always create a custom theme.

**Pull request merged**